client/landing/gutenboarding/components/create-site-error/index.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import { Icon, wordpress } from '@wordpress/icons';
+import { useI18n } from '@automattic/react-i18n';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Link from '../link';
+import './style.scss';
+import { createInterpolateElement } from '@wordpress/element';
+import { localizeUrl } from 'lib/i18n-utils';
+import ActionButtons from '../../components/action-buttons';
+import { Title, SubTitle } from '../../components/titles';
+interface Props {
+	linkTo: string;
+}
+
+const CreateSiteError: FunctionComponent< Props > = ( { linkTo } ) => {
+	const { __ } = useI18n();
+
```

Comment:
To address the [browser navigation point below](https://github.com/Automattic/wp-calypso/pull/43092/files#r439910944), I think we might need to clear the site store here when the component unmounts, e.g., 

```
	const { reset: resetSignupStore } = useDispatch( SITE_STORE );

	useEffect( () => resetErrors );

```

It should be okay to clear the entire site store, but we could also create an action to just reset the site errors in the site store actions file:

https://github.com/Automattic/wp-calypso/blob/bb8372d/packages/data-stores/src/site/actions.ts

What do you think?

Reply:
So this scenario:

/new/plans -> /new/create-site -> /new/create-site (error) -> click "Go Back" -> /new/plans -> click browser back button -> /new/create-site (error)

What was happening is the "Go back" wasn't a go back but a link to the plans page, hitting browser back would take you back to the error message. I think this actually makes a bit of sense, if you hit back again you'll end up back at the plans page.

I don't know if this is something that needs fixing (da3c356 has an attempt) and what is the desired outcome be if you select back while on the plans page after seeing an error?